### ISSUE TITLE
Criminal use of my Information

### ISSUE THREAD

**Issue by RuiFLourenco** [jsbin/jsbin#3102]
Good morning,

I would like to know why is my social security number (CPF:234.167.398-84 ) in the page
https://jsbin.com/nutokenuxu/1/edit?html,output

It is a crime to divulge this information since it can be used to criminal intent. Please remove it.

Thank you very much
Rui

**Comment by remy** [comment]
I'm not entirely sure why you're asking on github. I states clearly that the bin was created anonymously, so I have no answer for you. I'm guessing someone was trying to dox you? I don't know, sorry.

I have removed it though.

However, you've put your own social security number in this web site, and you'll need to get this information deleted too (along with the gist, I'm assuming you found the bin from…I did a quick google for your SSN).
